competitor.php

```diff
@@ -11,172 +11,101 @@
-nur Menschen, und so kann es unter Umständen schon mal passieren, dass
-dein Teilnehmerzettel bei einem Event vergessen wurde oder verloren
-gegangen ist.
-</p>
-
-<h2>Du wurdest aufgerufen</h2>
+Teilnehmer eines Events werden meistens in Gruppen eingeteilt, die du den ausgehängten Listen
+entnehmen kannst. Sobald dein Name oder deine Gruppe aufgerufen wird, gehe zum Scrambling
+Tisch und lege dein Puzzle auf den Zettel (Scoresheet), auf dem dein Name steht. Danach setzt
+du dich in den Wartebereich. Wenig später wird dich ein Judge aufrufen. Gehe zusammen mit ihm
+zu einem der Timertische und setze dich. Der Judge legt das Puzzle vor dir auf die Matte
+(Stackmat). Zu Beginn des Versuchs ist es mit einer Abdeckung verdeckt. Dein Judge wird dich
+nun fragen, ob du bereit bist ("Ready?"). Falls du zu diesem Zeitpunkt noch unsicher bist, was
+den weiteren Ablauf angeht (z.B. wenn du nicht weißt, wie der Timer funktioniert), zögere nicht
+den Schiedsrichter einfach zu fragen. Sobald du aber eindeutig bejahst ("Ja!"), dass du bereit
+bist, wird er die Abdeckung hochheben, womit die Inspection Phase beginnt.</p>
+
+<h2>Inspection und Beginn des Versuchs</h2>
 
 <p>
-Jetzt geht es los. Dein Name wurde aufgerufen. Es steht ein
-Schiedsrichter mit deinem gemischten Puzzle beim Wartebereich, um
-dich abzuholen. Gehe zu ihm und lasse dir weitere Anweisungen geben.
-Wenn nötig, sage ihm, dass das deine erste Competition ist und dass er dir bitte
-noch einmal erklären soll, wie es jetzt genau weiter geht.
-Mehr findest du im <a href="solve.php">Puzzle Lösen Tutorial</a>.
+Du hast nun bis zu 15 Sekunden Zeit, dir das Puzzle anzuschauen und den Anfang deiner Lösung
+zu planen. In dieser Zeit passiert folgendes: Du darfst das Puzzle in der Hand halten, aber
+keine Drehungen machen. Dein Judge wird dir nach 8 Sekunden ("8 seconds") und
+12 Sekunden ("12 seconds") die Zeit ansagen. Du kannst allerdings jederzeit in der Inspection
+Phase den Versuch starten. Dazu legst du das Puzzle wieder auf die Matte und startest den Timer,
+indem du beide Hände <i>flach</i> auf die Sensoren legst. Warte kurz bis die grüne Lampe
+leuchtet. Wenn du nun die Hände herunternimmst, läuft die Zeit und du kannst anfangen das Puzzle
+zu lösen. Spätestens nach den 15 Sekunden musst du den Timer gestartet haben.
 </p>
 
-<h2>Fertig? Hinsetzen!</h2>
+<h2>Ende des Versuchs</h2>
 
 <p>
-Normalerweise löst jeder Teilnehmer ein Puzzle fünf Mal (drei Mal für
-bestimmte Puzzle). Nachdem du dein Puzzle gelöst hast, musst du zurück
-in den Wartebereich. Nimm dein Puzzle nicht mit, sondern gib es dem
-Schiedsrichter, damit dieser es erneut mischen lassen kann.
+Wenn du das Puzzle gelöst hast, lege das Puzzle auf den Tisch und stoppe den Timer. Dazu musst du
+wieder die Hände <i>flach</i> auf die Sensorflächen legen. Achte darauf, dass du das Puzzle dabei
+nicht mehr berührst.
+
+Dein Judge wird nun überprüfen, ob das Puzzle gelöst ist und deine Zeit aufschreiben. Er wird dir
```

Comment:
I assume the paragraphs in the code here are supposed to be paragraphs in the HTML as well? If so then you are missing some "p" tags.